### PR TITLE
Add more flow types and flow scripts

### PR DESCRIPTION
**Pull request by TrySound**

### CODE REVIEW

**Comment by TrySound** [comment]
@souporserious Are you okay with this?

File: src/utils.js [hunk 1]
```diff
@@ -75,11 +107,11 @@ export function getElementSize(node) {
   }
 }
 
-export function getMeasurements(node) {
+export function getMeasurements(node: HTMLElement) {
   const rect = node.getBoundingClientRect()
   return {
-    x: rect.x,
-    y: rect.y,
+    x: rect.left,
```
Comment (souporserious) [inline src/utils.js:113]: Any particular reason not use `x` and `y` here?

**Comment by souporserious** [comment]
Just one question 😇, but other than that this looks great! Thank you so much 🙏

File: src/utils.js (hunk 1 above)
Comment (TrySound) [reply src/utils.js:113]: Flow does not describe this two. Seems like this type is IE/EDGE specific.
https://github.com/facebook/flow/blob/master/lib/dom.js#L3045-L3052

Comment (souporserious) [reply src/utils.js:113]: Ah I see! Thanks for letting me know 😁

**Comment by TrySound** [comment]
Let me rebase it

**Comment by souporserious** [comment]
Ah oops I just did! 🤦‍♂️ I hope I did it right.

**Comment by TrySound** [comment]
This would be good to merge before somebody start open PRs to rewrite branch

**Pull request merged**